src/components/ImageView/index.tsx

```diff
@@ -165,14 +140,14 @@ function ImageView({isAuthTokenRequired, url, fileName, onError}) {
     );
 
     const trackMovement = useCallback(
-        (e) => {
+        (e: MouseEvent) => {
```

Comment:
Same here:

```ts
    const trackMovement = useCallback(
        (e: MouseEvent) => {
            const mouseEvent = e as unknown as ReactMouseEvent;
            if (!isZoomed) {
                return;
            }

            if (isDragging && isMouseDown && scrollableRef.current) {
                const x = mouseEvent.nativeEvent.x;
                const y = mouseEvent.nativeEvent.y;
                const moveX = initialX - x;
                const moveY = initialY - y;
                scrollableRef.current.scrollLeft = initialScrollLeft + moveX;
                scrollableRef.current.scrollTop = initialScrollTop + moveY;
            }

            setIsDragging(isMouseDown);
        },
        [initialScrollLeft, initialScrollTop, initialX, initialY, isDragging, isMouseDown, isZoomed],
    );
```

Reply:
Well if it's well tested and it's working I believe you can change to remove `nativeEvent`, same for the other case `const isInsideImageView = scrollableRef.current.contains(e.nativeEvent.target);`.

Reply:
How about this?
```ts
    const trackMovement = useCallback(
        (event: MouseEvent) => {
            if (!isZoomed) {
                return;
            }

            if (isDragging && isMouseDown && scrollableRef.current) {
                const x = event.x;
                const y = event.y;
                const moveX = initialX - x;
                const moveY = initialY - y;
                scrollableRef.current.scrollLeft = initialScrollLeft + moveX;
                scrollableRef.current.scrollTop = initialScrollTop + moveY;
            }

            setIsDragging(isMouseDown);
        },
        [initialScrollLeft, initialScrollTop, initialX, initialY, isDragging, isMouseDown, isZoomed],
    );
```

Reply:
Same for the other function: 
```ts
    const trackPointerPosition = useCallback(
        (event: MouseEvent) => {
            // Whether the pointer is released inside the ImageView
            const isInsideImageView = scrollableRef.current?.contains(event.target as Node);

            if (!isInsideImageView && isZoomed && isDragging && isMouseDown) {
                setIsDragging(false);
                setIsMouseDown(false);
            }
        },
        [isDragging, isMouseDown, isZoomed],
    );
```